phpBB/phpbb/db/migration/data/v310/acp_prune_users_module.php

```diff
@@ -70,12 +72,7 @@ public function move_prune_users_module()
 		$acp_cat_users_id = (int) $this->db->sql_fetchfield('module_id');
 		$this->db->sql_freeresult($result);
 
-		if (!class_exists('\acp_modules'))
-		{
-			include($this->phpbb_root_path . 'includes/acp/acp_modules.' . $this->php_ext);
-		}
-		$module_manager = new \acp_modules();
-		$module_manager->module_class = 'acp';
-		$module_manager->move_module($acp_prune_users_id, $acp_cat_users_id);
+		$module_manager = $phpbb_container->get('module.manager');
```

Comment:
Get it from the system insted of global `$phpbb_container`

Extend `phpbb\db\migration\container_aware_migration` and then use `$this->container->`